src/engines/engine-decorator.js

```diff
@@ -42,8 +42,16 @@ class EngineDecorator extends FakeEventTarget implements IEngineDecorator {
           return this._listeners;
         }
         const activeDecorator = this._pluginDecorators.find(decorator => decorator.active);
-        // $FlowFixMe
-        return activeDecorator && prop in activeDecorator ? activeDecorator[prop] : obj[prop];
+        const isGetter = (object, property) => {
+          const descriptor = Object.getOwnPropertyDescriptor(object, property);
+          return descriptor && !!descriptor['get'];
+        };
+        const activeEngine = activeDecorator && prop in activeDecorator ? activeDecorator : obj;
```

Comment:
sounds like the proper name for this is just `target` as it can be the active decorator or the engine

Reply:
```suggestion
        const target = activeDecorator && prop in activeDecorator ? activeDecorator : obj;
```